samples/net/sockets/mt_http_server/README.rst

```diff
@@ -0,0 +1,62 @@
+.. _sockets-mt-http-server-sample:
+
+Socket Multithreaded HTTP Server
+################################
+
+Overview
+********
+
+The sockets/mt_http_server sample application for Zephyr implements a
+skeleton HTTP server using a BSD Sockets compatible API.
+This sample has similar functionality as :ref:`sockets-dumb-http-server-sample`
+except here we have support for multiple simultaneous connections and IPv6.
+Also the sample application has no compatibility with POSIX.
+This HTTP server example is very minimal and does not really parse an incoming
+HTTP request, just reads and discards it, and always serves a single static
+page.
+
+The source code for this sample application can be found at:
+:zephyr_file:`samples/net/sockets/mt_http_server`.
+
+Requirements
+************
+
+- :ref:`networking_with_host`
+- or, a board with hardware networking
+
+Building and Running
+********************
+
+Build the Zephyr version of the sockets/echo application like this:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/sockets/mt_http_server
+   :board: <board_to_use>
+   :goals: build
+   :compact:
+
+``board_to_use`` defaults to ``qemu_x86``. In this case, you can run the
```

Comment:
I don't think there's a default board any more, and you're requited to specify a board.

Reply:
Good point. I copied the readme from other sample, the invalid instructions are currently found in these samples
```
samples/net/sockets/big_http_download/README.rst
39:``board_to_use`` defaults to ``qemu_x86``. In this case, you can run the

samples/net/sockets/echo_async_select/README.rst
34:``board_to_use`` defaults to ``qemu_x86``. In this case, you can run the

samples/net/sockets/echo/README.rst
34:``board_to_use`` defaults to ``qemu_x86``. In this case, you can run the

samples/net/sockets/http_get/README.rst
37:``board_to_use`` defaults to ``qemu_x86``. In this case, you can run the

samples/net/sockets/dumb_http_server/README.rst
40:``board_to_use`` defaults to ``qemu_x86``. In this case, you can run the

samples/net/sockets/echo_async/README.rst
36:``board_to_use`` defaults to ``qemu_x86``. In this case, you can run the
```